shared/ui/core/src/test/java/com/ivy/ui/FormatMoneyUseCaseTest.kt

```diff
@@ -0,0 +1,52 @@
+package com.ivy.ui
+
+import androidx.compose.runtime.Composable
+import com.ivy.domain.features.BoolFeature
+import com.ivy.domain.features.FeatureGroup
+import com.ivy.domain.features.Features
+import com.ivy.ui.time.DevicePreferences
+import io.kotest.matchers.shouldBe
+import io.mockk.every
+import io.mockk.mockk
+import org.junit.Before
+import org.junit.Test
+import java.util.Locale
+
+class FormatMoneyUseCaseTest {
```

Comment:
Test all possible cases using TestParameter enum (inspect other tests written by me). The idea is to test multiple cases and verify that they're formatted correctly. 

enum class FormatTestCase(
  amount: Double,
  locale: Locale, 
  showDecimal: Boolean,
  expectedFormatted: String
)

Reply:
To do this you'll need RunWith(TestParameterInjector)